src/js/components/Collapsible/__tests__/Collapsible-test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import 'jest-styled-components';
+import { cleanup, render, fireEvent } from '@testing-library/react';
+
+import { Collapsible } from '..';
+import { Box } from '../../Box';
+import { Button } from '../../Button';
+import { Grommet } from '../../Grommet';
+import { Text } from '../../Text';
+
+describe('Collapsible', () => {
+  afterEach(cleanup);
+
+  test('open', () => {
+    const { container } = render(
+      <Grommet>
+        <Collapsible open>
+          <Box background="dark-1">Box Body</Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('onClick open default', () => {
+    const collapseOnClick = jest.fn();
+    let open = false;
+    const { container, rerender, getByText } = render(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            collapseOnClick();
+          }}
+          label="Expand"
+        />
```

Comment:
Could the Button be removed entirely and instead of changing the value of open in collapseOnClick open could be changed outside of the render function just by setting it to true. I think this would have the same effect and there wouldn't be a need for the button in the test code anymore